CMakeLists.txt

```diff
@@ -2,6 +2,9 @@
 #								#
 # Copyright (c) 2012-2016 Fidelity National Information		#
 # Services, Inc. and/or its subsidiaries. All rights reserved.	#
+#								#
+# Copyright (c) 2017 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
 #                                                               #
 #  Copyright (c) 2017 Finxact, LLC. and/or its subsidiaries.    #
```

Comment:
My ownly question is why the Finxact CC is being kept. The only interest Finxact had in it was spun off into YottaDB LLC so this is in effect mentioning our interest twice. If we do for some reason keep it, the YottaDB LLC mention should be last since copyrights are temporally ordered.